OpenRA.Mods.Common/Scripting/Global/DateTimeGlobal.cs

```diff
@@ -32,16 +32,18 @@ public int GameTime
 			get { return Context.World.WorldTick; }
 		}
 
-		[Desc("Converts the number of seconds into game time (ticks).")]
-		public int Seconds(int seconds)
+		[Desc("Converts the number of seconds into game time (ticks), measured in time passed in-game by default,",
+			"or by real time if `walltime` is true.")]
+		public int Seconds(int seconds, bool walltime = false)
```

Comment:
As mentioned earlier, i'd be a lot happier if we could find a way to avoid adding `walltime` if we can do it without hurting the usability of the timer feature.